CSS/basic.css

```diff
@@ -0,0 +1,32 @@
+.Robotofont {
+  font-family: "Roboto", sans-serif;
+}
+
+.Poppinsfont {
+  font-family: "Poppins", sans-serif;
+}
+
+.CreteRoundfont {
+  font-family: "Crete Round", serif;
+}
+
+.Interfont,
+.tecnoBTN > li,
+.nostyle > li {
+  font-family: "Inter", sans-serif;
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  min-height: 100%;
+  margin: 0%;
+}
+
+h1,
+p,
+ul {
+  width: 60%;
+}
```

Comment:
- [ ] Kindly make the width here `75%` to make the navbar goes to the left as the Figma design.

![image](https://user-images.githubusercontent.com/35535130/166123080-0b29e42f-69f7-4114-83eb-f72f6caa9c7e.png)